data/unihannover/unihannover.csv

```diff
@@ -101,3 +101,43 @@ Hannover U,2016,1182,10.1186/s12860-016-0116-6,FALSE,NA,NA,NA,NA
 Hannover U,2016,1322.87,10.1109/TPEL.2016.2537372,TRUE,NA,NA,NA,NA
 Hannover U,2016,1322.87,10.1109/TEMC.2016.2554463,TRUE,NA,NA,NA,NA
 Hannover U,2016,1020.50,10.1109/JPHOT.2016.2570001,FALSE,NA,NA,NA,NA
+Hannover U,2017,1820.7,10.1002/ece3.3531,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1552.95,10.1038/s41598-017-04539-2,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1552.95,10.1038/s41598-017-06542-z,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1552.95,10.1038/s41598-017-12267-w,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,662.76,10.1039/C7RA05071B,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1519.12,10.1063/1.4978266,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1333.77,10.1080/23311908.2017.1329191,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1392.3,10.1088/1367-2630/aa5c0c,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1703.73,10.1109/TEMC.2017.2666880,TRUE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1474.05,10.1109/TEMC.2017.2682643,TRUE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1806,10.1127/metz/2017/0853,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1877.14,10.1155/2017/9124254,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,2038.17,10.1186/s13006-017-0105-1,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1682.29,10.1371/journal.pone.0173580,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1646.2,10.1371/journal.pone.0176303,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1503.3,10.1371/journal.pone.0185310,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1825.46,10.2147/CIA.S125201,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1825.46,10.2147/IDR.S144299,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1524.6,10.2196/ijmr.7352,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1042.44,10.3390/atmos8090168,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1564.13,10.3390/en10020248,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1566.89,10.3390/en10101495,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1543.41,10.3390/en10111688,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1615.43,10.3390/ijerph14070825,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1697.09,10.3390/ijerph14080920,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1958.69,10.3390/ijms18071415,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,2012.13,10.3390/s17020345,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1333.23,10.3390/s17030517,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1966.77,10.3390/s17051118,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1909.23,10.3390/s17081886,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1484.96,10.3390/su9081470,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,373.53,10.3390/toxics5030015,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1537.37,10.3390/w9050359,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,1249.5,10.5194/acp-2017-134,FALSE,NA,NA,NA,NA,NA,NA
+Hannover U,2017,154.7,10.5194/ars-14-47-2016,FALSE,NA,NA,NA,NA,NA,NA
```

Comment:
@tullney: This one forms a duplicate with line 77, solution required

Reply:
:-( - we'll investigat and send a PR

Reply:
see #359